tools/roslaunch/src/roslaunch/substitution_args.py

```diff
@@ -46,6 +46,7 @@
 
 import rosgraph.names
 import rospkg
+import loader
```

Comment:
PEP 8 recommends absolute imports (see https://www.python.org/dev/peps/pep-0008/#imports). If you use the `from` syntax it needs to be absolute.

(Also I don't think any code within this repo uses relative imports currently?)